contrib/spring_boot2/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenTelemetry Spring Boot2 Starter
```

Comment:
I think this is a contrib SDK package because it depends on the SDK

Reply:
Good point. I have moved it under sdk_contrib